sdk/core/azure-core-serializer-json-gson/src/main/java/com/azure/core/serializer/json/gson/GsonJsonSerializer.java

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.serializer.json.gson;
+
+/*
+ * Thoughts on this class skipping the builder pattern?
```

Comment:
I'm wondering if it would be easier/more convenient to just have these classes take the pre-built serializer for a given module, a `Gson` class for gson JSON parsing, `ObjectMapper` for Jackson, etc.